workflow/metrics/metrics.go

```diff
@@ -216,11 +217,11 @@ func (m *Metrics) OperationPanic() {
 	m.errors[ErrorCauseOperationPanic].Inc()
 }
 
-func (m *Metrics) CronWorkflowSubmissionError() {
+func (m *Metrics) CronWorkflowError(cause ErrorCause) {
```

Comment:
We don't want reporting functions with parameters in this file. Create a new function for the new error please.

Reply:
Thanks. Updated.

> We don't want reporting functions with parameters in this file

Just curious - is there a reason behind this? Perhaps a style guide somewhere I may have missed?